driver/command/command.go

```diff
@@ -101,5 +119,62 @@ func (d *Driver) exec(op *driver.Operation) (driver.OperationResult, error) {
 		return driver.OperationResult{}, fmt.Errorf("Start of driver (%s) failed: %v", d.Name, err)
 	}
 
-	return driver.OperationResult{}, cmd.Wait()
+	if err = cmd.Wait(); err != nil {
+		return driver.OperationResult{}, fmt.Errorf("Command driver (%s) failed executing bundle: %v", d.Name, err)
+	}
+
+	result, err := d.getOperationResult(op)
+	if err != nil {
+		return driver.OperationResult{}, fmt.Errorf("Command driver (%s) failed getting operation result: %v", d.Name, err)
+	}
+
+	return result, nil
+}
+func (d *Driver) getOperationResult(op *driver.Operation) (driver.OperationResult, error) {
+	opResult := driver.OperationResult{
+		Outputs: map[string]string{},
+	}
+
+	if len(op.Outputs) == 0 {
```

Comment:
I guess this check is not useful as if `Outputs` is empty, ranging on it won't do anything?

Reply:
Not sure I follow , function returns if outputs is empty?

Reply:
Yes but just after you iterate on `op.Outputs`, so if it is empty it will be no-op right? And then return at the end of the function?

Reply:
I realized what you meant after I replied, I was just trying to be explicit about it, anyway that code has gone now it checks for default values.